vignettes/ggPMX-ggplot.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to use ggplot2 syntax with ggPMX"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use ggplot syntax with ggPMX}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We will create a simple ggPMX controller using the theophylline example:
+
+```{r, warning=FALSE, message=FALSE}
+
+library(ggPMX)
+library(ggplot2)
+
+ctr <- theophylline()
+```
+
+Each ggPMX plot is an object with class ggplot:
+
+```{r, warning=FALSE, message=FALSE}
+p <- ctr %>% pmx_plot_dv_ipred()
+class(p)
+```
+
+
+Therefore we can use usual ggplot2 syntax to customize a ggPMX plot.
+
+Let's look at some examples.
+
+1. Facet
+
+ggPMX :
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred(strat.facet= ~SEX)
+```
+
+ggplot:
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred() + facet_wrap(~SEX)
+```
+
+As we see in general we have the same chart, but ggPMX also automatically creates an appropriate title.
+
+2. Title
+
+ggPMX :
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred(labels=c(title="New title!"))
+```
+
+ggplot:
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred() + ggtitle("New title!")
+```
+
+3. X and Y axis labels
+
+ggPMX :
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
```

Comment:
If the `fig.height` and `fig.width` are common, perhaps you should add this in the overall rmarkdown header.

Reply:
Yes, will use one  knitr::opts_chunk$set call instead of set options in each chunk.